packages/core/src/2d/data/RenderData2D.ts

```diff
@@ -0,0 +1,14 @@
+import { Color, Vector2, Vector3 } from "@oasis-engine/math";
+
+export interface RenderData2D {
+  /** @internal */
+  vertexCount: number;
+  /** @internal */
+  positions: Vector3[];
+  /** @internal */
+  uvs: Vector2[];
+  /** @internal */
+  triangles: number[];
+  /** @internal */
+  color: Color;
+}
```

Comment:
internal class pr internface add @internal before `export interface RenderData2D` is enough.

It is recommended to name it spriteMesh，you will found is nearly our `Mesh`